upup/pkg/fi/cloudup/defaults.go

```diff
@@ -167,10 +167,10 @@ func ensureKubernetesVersion(c *kops.Cluster) error {
 }
 
 // FindLatestKubernetesVersion returns the latest kubernetes version,
-// as stored at https://storage.googleapis.com/kubernetes-release/release/stable.txt
+// as stored at https://dl.k8s.io/release/stable.txt
 // This shouldn't be used any more; we prefer reading the stable channel
 func FindLatestKubernetesVersion() (string, error) {
-	stableURL := "https://storage.googleapis.com/kubernetes-release/release/stable.txt"
+	stableURL := "https://dl.k8s.io/release/stable.txt"
```

Comment:
Not quite sure what's wrong here:
```
% curl https://storage.googleapis.com/kubernetes-release/release/stable.txt
v1.27.1

% curl https://dl.k8s.io/release/stable.txt                                
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Reply:
Try adding a `-L` to your curl

Reply:
We should ensure redirects are used anywhere `stableURL` is referenced.

Reply:
Thanks @rjsadow. 😁